codespell_lib/data/dictionary.txt

```diff
@@ -12240,7 +12249,7 @@ facorites->favorites
 facors->favors, fakers,
 facour->favour, favor,
 facourite->favourite, favorite,
-facourites->favuourites, favorites,
+facourites->favourites, favorites,
 facours->favours, favors,
```

Comment:
Same here, they clearly meant favour* so we probably shouldn't suggest the US version.

Reply:
Ok, i wanted to be consistent with the line above. I deleted the second suggestion in both lines